### PR TITLE
Update integration tests to use the new CLI

### PR DESCRIPTION
**Pull request by bamarni**
- Fix package_plugin.py on Windows
- Fix test_task.py:test_attach
- Update DC/OS installer URL used in tests to 1.12
- Update integration tests to use the new CLI

https://jira.mesosphere.com/browse/DCOS_OSS-4189

### CODE REVIEW

**Comment by bamarni** [comment]
@mesosphere-ci run linux integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run linux integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run linux integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run linux integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run linux integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run windows integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run windows integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run windows integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run windows integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run windows integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run windows integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run windows integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run linux integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run mac integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run linux integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run mac integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

File: python/lib/dcoscli/tests/integrations/test_task.py [hunk 1]
```diff
@@ -334,17 +336,17 @@ def test_exec_match_id_pattern():
...
     tty.setraw(master, when=termios.TCSANOW)
 
-    master.write("echo 'Hello World!'\r\n")
+    msg = "Hello World!\r\n"
+    master.write(msg)
     master.flush()
 
-    echo = proc.stdout.read(33).decode("utf-8")
-    assert echo == "echo 'Hello World!'\r\nHello World!"
+    assert proc.stdout.read(len(msg)).decode() == msg
```
Comment (klueska) [inline python/lib/dcoscli/tests/integrations/test_task.py]: You should actually see the message twice though...

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

File: python/lib/dcoscli/tests/integrations/test_task.py [hunk 2]
```diff
@@ -334,17 +336,18 @@ def test_exec_match_id_pattern():
...
 
-    master.write("echo 'Hello World!'\r\n")
+    msg = "Hello World!\n"
+    expected_output = "Hello World!\r\nHello World!\r\n"
+    master.write(msg)
     master.flush()
 
-    echo = proc.stdout.read(33).decode("utf-8")
-    assert echo == "echo 'Hello World!'\r\nHello World!"
+    assert proc.stdout.read(len(msg)*2+2).decode() == expected_output
```
Comment (armandgrillet) [inline python/lib/dcoscli/tests/integrations/test_task.py]: Could we add a comment regarding the `*2+2`?

**Review by armandgrillet — APPROVED** [review]

File: python/lib/dcoscli/tests/integrations/test_task.py (hunk 2 above)
Comment (bamarni) [reply python/lib/dcoscli/tests/integrations/test_task.py]: I had one but just removed it as suggested by @klueska, because it is just how cat works.

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

File: python/lib/dcoscli/tests/integrations/test_task.py (hunk 2 above)
Comment (klueska) [reply python/lib/dcoscli/tests/integrations/test_task.py]: No, this line should be:
```
assert proc.stdout.read(len(expected_output)).decode() == expected_output
```
As discussed on slack.

Comment (klueska) [reply python/lib/dcoscli/tests/integrations/test_task.py]: If you want to put a comment about the convesion of `\n' to `\r\n`, I woudl do it above line 345

Comment (bamarni) [reply python/lib/dcoscli/tests/integrations/test_task.py]: I had missed that part, just updated it.

**Pull request merged**